Tests/TensorFlowTests/LossTests.swift

```diff
@@ -16,6 +16,23 @@ import XCTest
 @testable import TensorFlow
 
 final class LossTests: XCTestCase {
+
```

Comment:
Remove extra line.

Reply:
@rxwei I'm sorry for these weird popups in my commits, I've found out that git has this nasty habit of trying to fix line endings and adding extra lines after functions start.